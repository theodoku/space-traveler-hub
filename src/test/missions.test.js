import Mission from '../components/mission';

describe('Mission component test', () => {
  const component = (
    <Mission
      id="1"
      mission="ilyass"
      description="atlassi"
    />
  );

  it('Testing Mission page rendering', () => {
    expect(component).toMatchSnapshot();
  });
});
