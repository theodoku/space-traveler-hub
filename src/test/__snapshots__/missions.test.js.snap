// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission component test Testing Mission page rendering 1`] = `
<Mission
  description="atlassi"
  id="1"
  mission="ilyass"
  reserved={false}
/>
`;
